package.json

```diff
@@ -21,23 +23,5 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject"
```

Comment:
After modifying the `.eslintrc.json`, add the following scripts and try to run these once and see there isn't any error - 
```
"lint": "eslint '**/*.ts' '**/*.tsx' '**/*.js'",
"prettier": "prettier --write */**/*.{js,jsx,ts,tsx,scss,html}",
```